app/controllers/home_controller.rb

```diff
@@ -1,11 +1,19 @@
 class HomeController < ApplicationController
   skip_before_action :authenticate_user!
   def index
-    @users = User.all
-
+    @users = User.all.order(:name).page(params[:page])
+    @posts = Post.paginate(page: params[:page], per_page: 10).order(:created_at)
```

Comment:
I guess we already have set `per_page` value in `post.rb`.
Try omitting it from here and test it again